config/parameters/sensors/base.txt

```diff
@@ -21,7 +21,7 @@ sensors.leader.gps.v.sigma  5.0 5.0 5.0
 
 sensors.leader.cdgps.model_range  1
 sensors.leader.cdgps.range        2000.0
-sensors.leader.cdgps.dr.sigma     0.1    0.1 0.1
+sensors.leader.cdgps.dr.sigma     0.01  0.01 0.01
```

Comment:
Just want to double check here, do we actually have a source for this? If not, we should open up a ticket in `psim` as well as flight software because the sensor noise covariances in both repositories are set off this number.